modules/instruments/README.md

```diff
@@ -0,0 +1,42 @@
+# Instruments Module
+
+## Purpose
+
+The instruments module is responsible for loading of instruments
+which are part of a study for data entry. It provides the routing
+to the appropriate instrument class to handle the page which is
+being handled.
+
+## Intended Users
+
+This module only does basic routing to pages which are used by data
+entry staff.
+
+## Scope
+
+The instruments module only loads existing instruments and provides
+data entry. It does not create or manage any metadata for an
+instrument.
+
+It also does not manage surveys in any way.
+
+## Permissions
+
+None. Individual instruments may provide their own permissions.
+
+## Configurations
+
+The `NDB_BVL_Instrument` class must be coded and put in the
+`project/instruments` directory, or a linst file must be created
+and put in the `project/instruments`.
+
+Individual instruments may contain their own configurations (such
+as requiring tables to be created or lookup tables.)
```

Comment:
```suggestion
as requiring tables to be created or lookup tables).
```

Reply:
should probably apply this again

Reply:
no.

Reply:
I think @PapillonMcGill is correct in this case based on some Googling. It's not a big deal though.

Reply:
Google also says that "color" and "neighbor" are correct spelling.

Reply:
Yes, as the parenthesis was opened inside a phrase, not to at the beginning of a new phrase, it as to be closed inside the phrase, so before the dot.
Not a big deal, as @johnsaigle  mentioned, this is why I just commented, not ask for change.